pkg/mic/mic_test.go

```diff
@@ -601,18 +601,20 @@ func NewMICTestClient(eventCh chan aadpodid.EventType,
 	podClient *TestPodClient,
 	nodeClient *TestNodeClient,
 	eventRecorder *TestEventRecorder, isNamespaced bool,
-	createDeleteBatch int64) *TestMICClient {
+	createDeleteBatch int64,
+	whiteListedIdentities map[string]bool) *TestMICClient {
```

Comment:
we should probably add a dedicated test the makes sure whitelisted identities are not deleted.